recipe/build.sh

```diff
@@ -4,10 +4,15 @@ python ${RECIPE_DIR}/brand_python.py
 
 if [ `uname` == Darwin ]; then
     export CFLAGS="-I$PREFIX/include $CFLAGS"
-    export LDFLAGS="-L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
+    export LDFLAGS="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
     sed -i -e "s/@OSX_ARCH@/$ARCH/g" Lib/distutils/unixccompiler.py
-    ./configure --enable-shared --enable-ipv6 --with-ensurepip=no \
-        --prefix=$PREFIX
+    ./configure \
+        --enable-ipv6 \
+        --enable-shared \
+        --prefix=$PREFIX \
+        --with-ensurepip=no \
+        --with-tcltk-includes="-I$PREFIX/include" \
+        --with-tcltk-libs="-L$PREFIX/lib -ltcl8.5 -ltk8.5"
```

Comment:
Is it possible that you have an older version of `conda-build`? There was a bug where it chopped stuff out of your environment variables not that long ago. It should be fixed in more recent versions.